libs/designsystem/calendar/src/calendar.component.ts

```diff
@@ -339,7 +339,7 @@ export class CalendarComponent implements OnInit, AfterViewInit, OnChanges {
     if (!this.activeMonth) return;
 
     const monthStart = startOfMonth(this.activeMonth);
-    const monthEnd = endOfMonth(this.activeMonth);
+    const monthEnd = add(startOfMonth(this.activeMonth), { weeks: 5 });
```

Comment:
Why do we always add five weeks to determine month-end?

Reply:
Good question @RasmusKjeldgaard ! It might be more appropriate to rename the variable to something other than `monthEnd,` because now the variable holds the value that corresponds to the first day of the current month + 5 weeks, which effectively makes the calendar show 6 weeks at all times :)

Reply:
Seems like some of these `const`s are only used for calculating the `totalDayCount` which I assume is always 42 now, right? So we probably dont need any fancy calculations now as there is always a grid of 7 weekdays * 6 rows of weeks.

Reply:
Nicely spottet! You're absolutely right :) I have removed all `const`s that were redundant :)